packages/netlify-cms-core/src/components/App/App.js

```diff
@@ -205,6 +208,7 @@ class App extends React.Component {
               to={defaultPath}
             />
             {hasWorkflow ? <Route path="/workflow" component={Workflow} /> : null}
+            {isGitBackend ? <Route path="/releases" component={Releases}/> : null }
```

Comment:
this seems to only work for github, is checking isGitBackend() enough to ensure we're in GitHub land ?